app/controllers/events_controller.rb

```diff
@@ -8,12 +8,28 @@ def search
     @event_list = Event.all
 
     if (params[:event])
-      @event         = Event.find_by(event_name_underscore: params[:event])
-      @event_rsvps   = @event.rsvps.order(status: :desc, updated_at: :desc)
-      @attending     = @event.rsvps.where(status: "Yes").length
-      @not_attending = @event.rsvps.where(status: "No").length
-      @no_reply      = @event.rsvps.where(status: "--").length
+      if (params[:event] == "messages")
```

Comment:
`if (params[:event])` is needed, otherwise whenever you land on `/events/search` it will look for params.